cosmo_tester/test_suites/cluster/conftest.py

```diff
@@ -34,20 +34,25 @@ def brokers_and_manager(cfy, ssh_key, module_tmpdir, attributes, logger):
 
 
 @pytest.fixture()
-def brokers_and_dbs_and_managers(cfy, ssh_key, module_tmpdir, attributes,
-                                 logger):
+def full_cluster(cfy, ssh_key, module_tmpdir, attributes,
+                 logger):
     for _vms in _get_hosts(cfy, ssh_key, module_tmpdir,
                            attributes, logger,
-                           broker_count=3, db_count=3, manager_count=2):
+                           broker_count=3, db_count=3, manager_count=2,
+                           pre_cluster_rabbit=True):
         yield _vms
 
 
 def _get_hosts(cfy, ssh_key, module_tmpdir, attributes, logger,
```

Comment:
maybe split this 500 line function? at least to making dbs and rabbits and managers in separate funcs?

Reply:
Refactored.